split.py

```diff
@@ -209,75 +207,7 @@ def merge_configs(main_config, additional_config):
 
 
 def configure_disassembler():
```

Comment:
I'd remove this function entirely and just do disassembler.configure directly

Reply:
OK Fixed